examples/ziti-end-to-end/README.md

```diff
@@ -0,0 +1,98 @@
+# Setting up an End-to-End Ziti network
+## Prerequisites:
+
+Ziti Binaries and relevant configs.
+
+[OpenZiti](https://github.com/openziti/ziti) contains downloads for the latest builds or you can build it yourself. There are many example configs [in the etc dir](https://github.com/openziti/ziti/tree/release-next/etc).
```

Comment:
nit: I prefer to link to the release branch "main" because the coherence and stability of "release-next" is as assured.